src/model/owned/table_type/entry.rs

```diff
@@ -164,8 +163,7 @@ impl ComplexEntry {
             out.write_u8(e.get_type()).unwrap();
 
             // Value
-            out.write_u32::<LittleEndian>(e.get_value() as u32)
-                .unwrap();
+            out.write_u32::<LittleEndian>(e.get_value() as u32).unwrap();
```

Comment:
Fixed on: https://github.com/gnieto/abxml-rs/pull/53

Originally, as I was writting to a Vec, I thought it was overkill returning a Result, but I think it's much better now. Thanks again for the feedback.